src/main/app/src/app/shared/material-form-builder/form-components/documenten-lijst/documenten-lijst.component.html

```diff
@@ -0,0 +1,75 @@
+  -->
+
+<div class="documenten-tonen-form-field">
+    <label [for]="data.id" class="label">
+        <div>{{data.label | translate}}</div>
+    </label>
+    <div [id]="data.id" class="value">
+        <mat-form-field *ngIf="!data.readonly" appearance="standard">
+            <mat-label>{{'filter' | translate}}</mat-label>
+            <input matInput (keyup)="applyFilter($event)">
+        </mat-form-field>
+        <div class="table-wrapper mat-elevation-z2">
+            <table mat-table [dataSource]="dataSource">
+                <ng-container matColumnDef="select">
+                    <th mat-header-cell *matHeaderCellDef></th>
+                    <td mat-cell *matCellDef="let document">
+                        <mat-checkbox (click)="$event.stopPropagation()"
+                                      (change)="change($event, document)"
+                                      [checked]="selection.isSelected(document)">
+                        </mat-checkbox>
+                    </td>
+                </ng-container>
+                <ng-container matColumnDef="titel">
+                    <th mat-header-cell *matHeaderCellDef> {{'titel' | translate}} </th>
+                    <td mat-cell *matCellDef="let document"> {{document.titel}} </td>
+                </ng-container>
+                <ng-container matColumnDef="documentType">
+                    <th mat-header-cell *matHeaderCellDef> {{'documentType' | translate}} </th>
+                    <td mat-cell *matCellDef="let document"> {{document.documentType}} </td>
+                </ng-container>
+                <ng-container matColumnDef="status">
+                    <th mat-header-cell *matHeaderCellDef> {{'status' | translate}} </th>
+                    <td mat-cell *matCellDef="let document"> {{'informatieobject.status.' + document.status | translate}} </td>
+                </ng-container>
+                <ng-container matColumnDef="versie">
+                    <th mat-header-cell *matHeaderCellDef> {{'versie' | translate}} </th>
+                    <td mat-cell *matCellDef="let document"> {{document.versie}} </td>
+                </ng-container>
+                <ng-container matColumnDef="auteur">
+                    <th mat-header-cell *matHeaderCellDef> {{'auteur' | translate}} </th>
+                    <td mat-cell *matCellDef="let document"> {{document.auteur}} </td>
+                </ng-container>
+                <ng-container matColumnDef="creatiedatum">
+                    <th mat-header-cell *matHeaderCellDef> {{'creatiedatum' | translate}} </th>
+                    <td mat-cell *matCellDef="let document"> {{document.creatiedatum}} </td>
+                </ng-container>
+                <ng-container matColumnDef="bestandsomvang">
+                    <th mat-header-cell *matHeaderCellDef> {{'grootte' | translate}} (MB)</th>
+                    <td mat-cell *matCellDef="let document"> {{document.bestandsomvang / 1048576 | number}} </td>
```

Comment:
magic number

En ik vroeg me af of hier ook die factor 4/3 van de base64 encoding (tijdens mail transport) in verwerkt zou moeten zitten. Maar het gaat hier niet noodzakelijk om een selectie voor het mailen, denk ik?